neasqc_wp61/6_Beta.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo 'This script classifies examples using beta_1 model.'
+
+
+while getopts "t:v:k:o:" flag
+do
+    case "${flag}" in
+        t) train=${OPTARG};;
+        v) test=${OPTARG};;
+        k) IFS=' ' read -r -a k <<< "${OPTARG}";;
+        o) output=${OPTARG};;
+    esac
+done
+
+echo "train: $train";
+echo "test: $test";
+echo "K values for KNN algorithm: ${k[*]}";
+echo "output: $output";
+
+echo "running beta_1"
+python3.10 ./data/data_processing/use_beta_1.py -tr ${train} -te ${test} -k ${k[*]} -o ${output}
```

Comment:
/!\ Here in the set_up_exp PR, the beta folder + beta files are renamed. It's not the case for the new general_quantum_distance PR.

Reply:
\+ Need to add the PCA arg